app/routes/goal_routes.py

```diff
@@ -1 +1,83 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, jsonify
```

Comment:
```suggestion
from flask import Blueprint, request, Response
```
We do not need to use `jsonify` in our API because Flask will convert a list or dictionary into JSON that we send back as a response. 

There are times when we need to create a `Response` object manually in order to set the mimetype to JSON and the status code to 204 (for our routes that don't return a response body, see line 44 in the PUT route in this file). 

Anywhere you use `jsonify` should be removed.